conf/crunchy-exporter/group.template

```diff
@@ -0,0 +1,33 @@
+root:x:0:
```

Comment:
Do we need this file?  I know `common_lib.sh` has the `ose_hack` function that references that file, but don't think we'll be using that here.

Can we therefore leave out this file and remove that unneeded function?

Reply:
Ok, thanks for that info. I've removed the template files and initial testing confirms the container is working as expected.